registry/handlers/manifests.go

```diff
@@ -476,3 +712,12 @@ func (imh *manifestHandler) DeleteManifest(w http.ResponseWriter, r *http.Reques
 
 	w.WriteHeader(http.StatusAccepted)
 }
+
+// annotatedTag will annotate OCI tags by prepending a string, and leave docker
+// tags unmodified.
+func (imh *manifestHandler) annotatedTag(oci bool) string {
```

Comment:
sigh..  ok let me push the fix to the commit to finish making it work... 

then add a commit that takes out the namespace "feature" from this PR